app/plugin_api.go

```diff
@@ -914,3 +914,59 @@ func (api *PluginAPI) PluginHTTP(request *http.Request) *http.Response {
 	api.app.ServeInterPluginRequest(responseTransfer, request, api.id, destinationPluginId)
 	return responseTransfer.GenerateResponse()
 }
+
+func (api *PluginAPI) CreateSlashCommand(cmd *model.Command) (*model.Command, *model.AppError) {
```

Comment:
Nit: We haven't traditionally used the term `SlashCommand` in the code, but just `Command`. Is now the time to diverge?

Reply:
Yep, changing it back.